metabase/metabase
# Add snowplow tracking

Epic https://github.com/metabase/metabase/issues/18291

This PR adds snowplow tracking in a similar way to google analytics. This means that:
- Snowplow tracker is only enabled when the user has enabled anonymous tracking
- It tracks only existing pageview events
- `CSP` header allows requests to `sp.metabase.com` only when anonymous tracking is enabled

How to test:
- Change `isProduction` to true
- Open the home page
- Navigate to other page
- Check that a request to `sp.metabase.com` has been fired, it contains data, and it is successful
